x-pack/plugins/spaces/server/config.ts

```diff
@@ -54,6 +54,7 @@ export const ConfigSchema = schema.object({
       defaultValue: true,
     }),
   }),
+  forceSolutionVisibility: schema.boolean({ defaultValue: false }),
```

Comment:
__note:__ I believe we should put it under `experimental` and restrict only to `traditional` offering to have less concerns when we decide to drop it:
```ts
  experimental: offeringBasedSchema({
    traditional: schema.object({
      forceSolutionVisibility: schema.boolean({ defaultValue: false }),
    }),
  }),
```

Reply:
Good suggestion. Addressed in https://github.com/elastic/kibana/pull/191743/commits/e7b89d1574630bc93caec66d4cb5a062b7c0666d